src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -1,44 +1,44 @@
 package edu.kis.vh.nursery;
 
-import edu.kis.vh.nursery.list.IntLinkedList;
+import edu.kis.vh.nursery.storage.IntLinkedList;
+import edu.kis.vh.nursery.storage.IntStorageInterface;
 
 /**
  * @author modyfikacji Damian Golema
  *
  */
 public class DefaultCountingOutRhymer 
 {
-	private IntArrayStack ias;
-	private IntLinkedList ill = new IntLinkedList();
-	
+	private IntStorageInterface isi = null;
+
 	public DefaultCountingOutRhymer() 
 	{
-		this.ias = null;

```

Comment:
ERROR: The default constructor creates an uninitialized instance! Should be implemented or removed